webui/src/layout/IfLoggedIn.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { useCurrentIdentityQuery } from './CurrentIdentity.generated';
+
+type Props = { children: React.ReactNode };
```

Comment:
If you make that a function, it would avoid rendering the child nodes if the user is not logged in

```suggestion
type Props = { children: () => React.ReactNode };
```